recipe/build.sh

```diff
@@ -1,18 +1,17 @@
 #!/bin/bash
 
-export LDFLAGS="-L${PREFIX}/lib"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
 export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
 
 if [ "$(uname)" == "Darwin" ];
 then
-    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
-    if [ $TRAVIS ];
-    then
-        brew uninstall gmp
-    fi
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib"
+elif [ "$(uname)" == "Linux" ];
+then
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath=${PREFIX}/lib"
```

Comment:
Why repeat the LDFLAGS code here? Actually, `build.sh` will only ever run on either Darwin or Linux, so....

Reply:
So they are actually slightly different. After `rpath`, Linux uses `=` and macOS uses `,`. Admittedly a bit subtle and seemingly needless difference. Unfortunately it is a difference we need to distinguish between.

Reply:
Oh, I missed that. Oh well.